src/common/mfu_daos.c

```diff
@@ -148,7 +154,9 @@ static int daos_parse_path(
         } else {
             strncpy(path, dattr.da_rel_path, path_len);
         }
-    } else if (strncmp(path, "daos:", 5) == 0) {
+    /* da_no_prefix is only used when only daos paths are expected,
+     * so if above parsing fails, then it is an error */
+    } else if (strncmp(path, "daos:", 5) == 0 || daos_no_prefix) {
```

Comment:
@dsikich We might consider allowing the serialize tools to use a daos:// prefix. Maybe not _**require**_ the prefix, but just _**allow**_ it in case a user does pass it.